src/templates/agent_detail.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% set active_page = 'agents' %}
+{% block content %}
+<div class="center">
+    Agent Name: {{ agent.name }}<br>
```

Comment:
Is it a good practice in frontend to use this many `<br>` elements? Can we replace it using `<p>` or row block for each property?

Reply:
By "row block" I assume you mean putting it in a table?
I experimented with both. The table approach felt pretty heavy, but perhaps with some additional styling it could work. I think the `<p>` would need quite a bit of css help as well, as it looks pretty weird natively:
![image](https://user-images.githubusercontent.com/1255513/233399499-fc72ee72-6228-4c5a-b86b-fc1e9c4abf63.png)

We've used `<br>` in the other templates as well, so while I agree that we have a lot of work to make this prettier and I make no claims of being a good gui designer, I think it might make more sense to go in later and take a consistent approach to updating all of them at once.

Reply:
No, I don’t mean tables. You can you containers and rows inside them (like `<div class=“row”`. You can include the vanilla css file (by using its link), so you don’t have to write any css file for it yourself

Reply:
Here is the doc on how to apply it (the Hotlink section)
https://vanillaframework.io/docs
Actually, it’s just one line of code

Reply:
Yeah, I definitely like the idea of moving to vanilla, but I think it's going to require a little more than just importing it and using this one element. There seem to be some side effects:
![image](https://user-images.githubusercontent.com/1255513/233456959-abe4f258-e06a-4179-a786-28776bf396c6.png)

We do have a jira issue for looking at converting to vanilla though, and adopting a more ubuntu-like theme. Or also, there's a chance still that we might replace the UI completely with something flutter based, so it may not matter eventually if that happens.

Reply:
Okay, you’re right, and we can keep it this way for now but rewrite it in future when we have a specific task for improving frontend design